src/arrays.js

```diff
@@ -1,61 +1,95 @@
 const getNthElement = (index, array) => {
-  // your code here
-};
+  if (index<=(array.length -1)) {
+    return array[index];
+  }
+  else {return array[(index%array.length)];
+}
```

Comment:
No need for the `else` statement here as `return` exits the function (also spacing and line breaks can make code a lot more readable):

```js
if (index <= (array.length - 1) {
  return array[index];
}

return array[index % array.length];
```